<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainShiftModule</class>
 <widget class="qMRMLWidget" name="BrainShiftModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>861</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Moving Image:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="qMRMLNodeComboBox" name="referenceVolume">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pick the output from the registration (the &lt;span style=&quot; font-weight:600;&quot;&gt;warped&lt;/span&gt; / moving image)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="SlicerParameterName" stdset="0">
      <string>referenceVolume</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Fixed Image:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="qMRMLNodeComboBox" name="backgroundVolume">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the fixed image from registration&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="hideChildNodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="interactionNodeSingletonTag">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Transformation:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="qMRMLNodeComboBox" name="transformNode">
     <property name="nodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="hideChildNodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="interactionNodeSingletonTag">
      <string notr="true"/>
     </property>
     <property name="SlicerParameterName" stdset="0">
      <string>transformNode</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output Volume:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="qMRMLCheckableNodeComboBox" name="displacementMagnitudeVolume">
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Compute Mapping</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Load Volume:</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="qMRMLNodeComboBox" name="MRMLReplacementVolume">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="hideChildNodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="interactionNodeSingletonTag">
      <string notr="true">Singleton</string>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Color Map:</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="qMRMLColorTableComboBox" name="colorMapSelector"/>
   </item>
   <item row="26" column="1">
    <widget class="QPushButton" name="loadDisplacementVolumeButton">
     <property name="text">
      <string>Load Volume</string>
     </property>
    </widget>
   </item>
   <item row="30" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="32" column="0">
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="33" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Select Landmarks to Visualize:</string>
     </property>
    </widget>
   </item>
   <item row="34" column="1">
    <widget class="QPushButton" name="ConvertTagFCSVButton">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Convert .tag File</string>
     </property>
    </widget>
   </item>
   <item row="35" column="1">
    <widget class="ctkCheckableComboBox" name="LandmarkSelectorComboBox"/>
   </item>
   <item row="36" column="1">
    <widget class="QPushButton" name="LoadExpertLabelsButton">
     <property name="text">
      <string>Load Landmarks for Visualization</string>
     </property>
    </widget>
   </item>
   <item row="37" column="0">
    <widget class="ctkCheckBox" name="enableHoverDisplayCheckbox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Enable Cursor DIsplay (mm)</string>
     </property>
    </widget>
   </item>
   <item row="38" column="0">
    <widget class="QCheckBox" name="enableUsBorderDisplay">
     <property name="whatsThis">
      <string>Controls the display of ultrasound borders</string>
     </property>
     <property name="accessibleName">
      <string>ultrasound border display checkbox</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Display US border</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="39" column="0">
    <widget class="QCheckBox" name="displayLandmarks">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Display Landmarks</string>
     </property>
    </widget>
   </item>
   <item row="40" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="41" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="42" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="43" column="0">
    <widget class="QLabel" name="opacitySliderLabel">
     <property name="text">
      <string>Opacity</string>
     </property>
    </widget>
   </item>
   <item row="43" column="1">
    <widget class="QSlider" name="opacitySlider">
     <property name="whatsThis">
      <string>Opacity slider</string>
     </property>
     <property name="accessibleName">
      <string>Opacity slider</string>
     </property>
     <property name="accessibleDescription">
      <string>this slider adjusts the opacity of the mri volume</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="45" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="46" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Threshold Slider:</string>
     </property>
    </widget>
   </item>
   <item row="46" column="1">
    <widget class="ctkDoubleRangeSlider" name="thresholdSlider">
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="maximumValue">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="47" column="1">
    <widget class="QDoubleSpinBox" name="thresholdMinSpinBox">
     <property name="specialValueText">
      <string>0.01</string>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="48" column="1">
    <widget class="QDoubleSpinBox" name="thresholdMaxSpinBox">
     <property name="specialValueText">
      <string>Maximum Threshold</string>
     </property>
    </widget>
   </item>
   <item row="49" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkCheckableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
